registry/storage/driver/hdfsweb/fs.go

```diff
@@ -0,0 +1,554 @@
+// Refer to github.com/vladimirvivien/gowfs
+
+package hdfsweb
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+)
+
+const (
+	webHdfsVer string = "/webhdfs/v1"
+)
+
+// FileSystem maps fields and functions to HDFS's FileSystem class.
+type FileSystem struct {
```

Comment:
Be more explicit with this comment. This is not only mapping the HDFS filesystem class, but it is also implementing over http.